spec/clientCalc.spec.mjs

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-undef */
+import { calculate } from '../src/client/utils/calculate.mjs';
+
+describe('addition', function() {
+    it('correctly adds', function() {
```

Comment:
The test names are a little vague, `it('should return 2 for 1+1', ...` would be a little better